torch/csrc/jit/codegen/cuda/ops/composite.cpp

```diff
@@ -29,6 +29,13 @@ ForwardDropoutResult dropout(TensorView* x, Val* prob, Val* scale) {
 
   auto rand_vals = randlike(x);
   auto mask = lt(rand_vals, prob);
+  // p == 0.0, set mask as False so everything is dropped.
```

Comment:
Is it needed? `lt(rand_vals, 0.0)` should always be all False because there shouldn't be any negative values generated by `randlike`.

Reply:
Good catch~
No it's not. Added it earlier for debug when I forgot we have `p` flipped :face_with_spiral_eyes: